### PR TITLE
BCN 04/19 - Katy Cooney

### PR DESCRIPTION
**Pull request by katharinecooney**
I had a couple of questions which I listed in the celebrityRoutes file! I wasn't able to get the /new GET path to work, so I have no idea of my /celebrities POST route works!

### CODE REVIEW

**Comment by katharinecooney** [comment]
I JUST FIGURED OUT WHY THE /NEW PATH WAS NOT WORKING! AHHHHHHH!!!!!!!!!!!!!!!!!!!!!

**Comment by tawebbcn** [comment]
> I JUST FIGURED OUT WHY THE /NEW PATH WAS NOT WORKING! AHHHHHHH!!!!!!!!!!!!!!!!!!!!!

**Comment by tawebbcn** [comment]
Too late jaja

File: mongoose-movies/app.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+mongoose.connect(`mongodb://localhost/${dbName}`,{
+  keepAlive: true,
+  userNewUrlParser: true,
+  reconnectTries: Number.MAX_VALUE
+});
+
+
+
+var indexRouter = require('./routes/index');
+var usersRouter = require('./routes/users');
```
Comment (tawebbcn) [inline mongoose-movies/app.js:18]: I think you don't need the users route

File: mongoose-movies/app.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+app.set('view engine', 'hbs');
+
+app.use(logger('dev'));
+app.use(express.json());
+app.use(express.urlencoded({ extended: false }));
+app.use(cookieParser());
+app.use(express.static(path.join(__dirname, 'public')));
+
+app.use('/', indexRouter);
+app.use('/users', usersRouter);
```
Comment (tawebbcn) [inline mongoose-movies/app.js:33]: Or here

File: mongoose-movies/views/celebrities/new.hbs [hunk 3]
```diff
@@ -0,0 +1,7 @@
+<h2></h2>
```
Comment (tawebbcn) [inline mongoose-movies/views/celebrities/new.hbs:1]: `h2`empty?

File: mongoose-movies/views/celebrities/show.hbs [hunk 4]
```diff
@@ -0,0 +1,8 @@
+<h2>Details</h2>
+{{#each celeb}}
+  {{this.name}}
```
Comment (tawebbcn) [inline mongoose-movies/views/celebrities/show.hbs:3]: Don't forget the tags! These lines are nothing to html

File: mongoose-movies/views/layout.hbs [hunk 5]
```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>{{title}}</title>
+    <link rel='stylesheet' href='/stylesheets/style.css' />
```
Comment (tawebbcn) [inline mongoose-movies/views/layout.hbs:5]: Don't forget the meta tag for the viewport configuration, is in the cheatsheet. And very important

File: mongoose-movies/models/CelebrityModel.js [hunk 6]
```diff
@@ -0,0 +1,6 @@
+const mongoose = require('mongoose');
```
Comment (tawebbcn) [inline mongoose-movies/models/CelebrityModel.js:1]: I would create the schema and the model in the same file

**Review by tawebbcn — COMMENTED** [review]
Good! I think you could have some file cleaner and be careful with html

**Pull request closed without merging**